devices/tuya.js

```diff
@@ -1798,9 +1798,23 @@ module.exports = [
         model: 'TS0004',
         vendor: 'TuYa',
         description: 'Smart light switch - 4 gang with neutral wire',
-        extend: extend.switch(),
-        exposes: [e.switch().withEndpoint('l1'), e.switch().withEndpoint('l2'), e.switch().withEndpoint('l3'),
-            e.switch().withEndpoint('l4')],
+        fromZigbee: [fz.on_off, fz.moes_power_on_behavior, fz.tuya_power_on_behavior, fz.tuya_backlight_switch,
+            fz.tuya_indicate_switch, fz.ignore_basic_report],
+        toZigbee: [tz.on_off, tz.moes_power_on_behavior, tz.tuya_power_on_behavior, tz.tuya_backlight_switch,
+            tz.tuya_indicate_switch],
+        exposes: [
+            e.power_on_behavior(),
+            exposes.switch().withState('backlight_switch', true, ''),
+            exposes.enum('indicate_switch', ea.ALL, ['off', 'switch', 'position']),
```

Comment:
I don't understand what this options does, can you add a description to it? e.g.

```js
exposes.enum('indicate_switch', ea.ALL, ['off', 'switch', 'position']).withDescription('blabla')
```

Reply:
These options present in tuya app, but changing their states doesn't affect device. I think there are similar devices which have the same model and they uses these options.

<details>
<summary>Tuya app</summary>
<img src="https://user-images.githubusercontent.com/16994749/171280630-ffeeded6-ad38-4ef9-a6ec-97d61220da88.png" width=250 />
</details>

Backlight turns on automatically when the device is powered.

As it uses 4 gang switch icon in tuya, maybe this devices uses options:
https://www.aliexpress.com/item/1005004218695401.html
<details>
<summary>icon</summary>
<img src="https://user-images.githubusercontent.com/16994749/171281580-c0e998a5-533e-4493-8727-a69ea28eff67.png" width=100 />
</details>

Of course, I can remove them if it's necessary

Reply:
If it doesn't do anything and it is unclear what it should do I propose to remove it (it only creates confusion otherwise)

Reply:
Done, https://github.com/Koenkk/zigbee-herdsman-converters/pull/4301/commits/7c7789c6b848906b35ed05dabc0300acef69f0a4